tasks/main.yml

```diff
@@ -35,6 +35,39 @@
         group: "{{ bind9_group }}"
         mode: 0644
 
+- name: Check if AppArmor is installed
+  become: true
+  shell: |
+    which aa-enabled
+  ignore_errors: true
+  register: app_armor_installed
+
+- name: Configure AppArmor if installed
+  when: app_armor_installed.rc == 0
+  block:
+    - name: Check if AppArmor enabled
+      become: true
+      shell: aa-enabled
+      register: app_armor
+      ignore_errors: true
+
+    - name: Extend AppArmor profile
+      become: true
+      shell: |
+        APP_ARMOR_LOCAL_HOME="/etc/apparmor.d/local"
+        FILE_NAME=$(which named | sed 's/\//./2g')
```

Comment:
This is done cause seems like the file name is based on where the binary is installed. Since this can be different based on os and preferences. I use which to construct the file name. So if `named` binary is in `/usr/sbin/named` the filename will be `usr.sbin.named`